composer.json

```diff
@@ -10,27 +10,22 @@
     "spryker/doctrine-inflector": "^1.0.0",
     "spryker/glue-application-extension": "^1.1.0",
     "spryker/kernel": "^3.30.0",
-    "spryker/symfony": "^3.0.0",
-    "spryker/twig": "^3.0.0",
-    "spryker/zend": "^2.0.0",
-    "symfony/console": "^3.4.0 || ^4.0.0",
-    "symfony/filesystem": "^3.4.0 || ^4.0.0",
-    "symfony/finder": "^3.4.0 || ^4.0.0",
-    "symfony/yaml": "^3.4.9 || ^4.0.0",
+    "symfony/console": "^3.4.0 || ^4.0.0 || ^5.0.0",
+    "symfony/filesystem": "^3.4.0 || ^4.0.0 || ^5.0.0",
+    "symfony/finder": "^3.4.0 || ^4.0.0 || ^5.0.0",
+    "symfony/yaml": "^3.4.9 || ^4.0.0 || ^5.0.0",
     "twig/twig": "^1.34 || ^2.0.0",
     "zendframework/zend-filter": "^2.8"
   },
   "require-dev": {
-    "codeception/codeception": "^2.4.1",
+    "codeception/codeception": "*",
```

Comment:
Are you sure we shouldnt keep semver version here? Future majors could break it all again